README.md

```diff
@@ -286,6 +302,35 @@ It will also exclude any files with the extension `.spec.js`.
 **Note:** Be wary of automatic OS glob expansion when specifying include/exclude globs with the CLI.
 To prevent this, wrap each glob in single quotes.
 
+### Including files within `node_modules`
+
+We always add `**/node_modules/**` to the exclude list, even if not specified in the config.
+You can override this by setting `--exclude-node-modules=false`.
+
+For example, in the following config, `"excludeNodeModules: false"` will prevent `node_modules` from being added to the exclude rules. 
+The set of include rules then restrict nyc to only consider instrumenting files found under the `lib/` and `node_modules/@my-org/` directories.
+The exclude rules then prevent nyc instrumenting anything in a `test` folder and the file `node_modules/@my-org/something/unwanted.js`.
+
+```json
+{
+  "nyc": {
+    "all": true,
+    "include": [
+      "lib/**",
+      "node_modules/@my-org/**"
+    ],
+    "exclude": [
+      "node_modules/@my-org/something/unwanted.js",
+      "**/test/**"
+    ],
+    "excludeNodeModules": false
+  }
+}
+```
+
+**Note:** You can also include `node_modules` files with a negated exclude rule `!**/node_modules/@my-org`.
```

Comment:
Instead of `You can also include`, maybe `An alternative method to include node_modules files is with ...`?

Reply:
Actually on further thought maybe don't even mention the negated exclude for node_modules?  I feel like now that we have `excludeNodeModules` boolean this is just the wrong way to accomplish the goal.

Reply:
True, and anyone already using can continue to without approval from the docs.  I'll get rid of it.